cli/cmd/volume.go

```diff
@@ -0,0 +1,168 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+const (
+	helpVolumeCmd       = "Gluster Volume Management"
```

Comment:
As purely a personal preference, unless these were used at multiple places, I'd place them in the declaration of `cobra.Command` itself.